app/views/raiserepair/search-property.html

```diff
@@ -14,43 +14,21 @@ <h1 class="heading-large">
         <span class="heading-secondary">Request a repair</span>
       </h1>
 
-      <form action="/raiserepair/book-appointment" method="post" class="form">
+      <form action="/raiserepair/search-property-results" method="post" class="form">
         <div class="form-group">
           <h2 class="heading-medium">
              What is your address?
           </h2>
 
           <div class="form-group">
             <label class="form-label" for="postcode">
-              Search for your address by postcode
+              Postcode
             </label>
-            <input class="form-control" id="postcode" type="text" name="postcode">
-            <a class="button" id="address-selection-button" data-target="address-selection">Find my address</a>
+            <input class="form-control" id="postcode" type="text" name="postcode" required>
```

Comment:
If there's an easy way to put the previously entered postcode here, that might make for a better demo, but no worries if it's fiddly.